lib/google/serverless/exec.rb

```diff
@@ -546,13 +562,21 @@ def service_from_config
       end
   
       def default_project
-        result = Util::Gcloud.execute \
+        result = Exec::Gcloud.execute \
           ["config", "get-value", "project"],
           capture: true, assert: false
         result.strip!
         raise NoDefaultProject if result.empty?
         result
       end
+
+      def default_product
+        if File.file?("/.app.yaml")
```

Comment:
This should just be `"app.yaml"`. If you start a file path with a slash, it's assumed to be in the root directory.